packages/node/src/node.ts

```diff
@@ -338,10 +388,15 @@ export class Node {
     const isProtocolMessage = (msg: NodeTypes.NodeMessage) =>
       msg.type === NODE_EVENTS.PROTOCOL_MESSAGE_EVENT;
 
-    const isExpectingResponse = (msg: NodeMessageWrappedProtocolMessage) =>
-      this.ioSendDeferrals.has(msg.data.processID);
+    const isFinishedMessage = (msg: NodeTypes.NodeMessage) =>
+      msg.type.endsWith("FinishedEvent");
+
+    const isExpectingResponse = (
+      msg: NodeMessageWrappedProtocolMessage | NodeMessageWrappedFinMessage
```

Comment:
okay looks like its being used in a few places, so may be worth adding this as a union type. i can make the change in follow on PRs though so its not blocking, but dont want the types package to have to be published twice